regxmllib/src/main/cpp/com/sandflow/smpte/regxml/definitions/DefinitionVisitor.h

```diff
@@ -84,6 +84,13 @@ namespace rxml {
 
 	public:
 
+#if defined(__clang__)
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Woverloaded-virtual"
+#elif defined(__GNUC__)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Woverloaded-virtual"
+#endif
```

Comment:
I am not getting these warnings in `gcc (Ubuntu 5.4.0-6ubuntu1~16.04.10) 5.4.0 20160609`